code/obj/decal/poster.dm

```diff
@@ -1145,3 +1139,97 @@
 				clear_banner()
 			else
 				return
+
+/obj/decal/poster/flag
+	name = "flag"
+	desc = "Neat!"
+	icon = 'icons/obj/decals/flag.dmi'
+	icon_state = "blank"
+	var/obj/item/flag/starting_flag = /obj/item/flag
```

Comment:
this var shouldn't be typed since it's not an instance, also should have a doc comment indicating it's a type var